docs/docs/how-tos/nebari-kubernetes.mdx

```diff
@@ -0,0 +1,167 @@
+---
+id: nebari-kubernetes
+title: How to deploy Nebari on pre-existing infrastructure
+description: Deploying Nebari on an existing Kubernetes infrastructure
+---
+
```

Comment:
@iameskild do you mind having a look into this to see if I forgot anything when moving this stuff?